google/cloud/speech/v1/cloud_speech_gapic.yaml

```diff
@@ -77,7 +77,10 @@ interfaces:
     long_running:
       return_type: google.cloud.speech.v1.LongRunningRecognizeResponse
       metadata_type: google.cloud.speech.v1.LongRunningRecognizeMetadata
-      polling_interval_millis: 20000
+      initial_poll_delay_millis: 3000
+      poll_delay_multiplier: 1.5
+      max_poll_delay_millis: 45000
+      total_poll_timeout_millis: 86400000
```

Comment:
Please note, that the MaxPollDelayMillis is used in exactly same manner as maxretrydelay is used for retries: it is max bound for the randomized value [0, max_poll_delay), so the expected value will be 45000/2 = 22500 milliseconds.